corehq/apps/reports/templates/reports/filters/single_option.html

```diff
@@ -1,6 +1,21 @@
 {% extends 'reports/filters/base.html' %}
 {% load hq_shared_tags %}
 {% block filter_content %}
+    <div class="reports-filters-single-option">
```

Comment:
Instead of giving the filter a unique id and initializing it immediately with inline javascript, changed this to assign a class to this type of filter and then initialize all `.reports-filters-single-option` in `filters.js` after the page is loaded. I'm imagining `filters.js` growing to encompass all reporting filters.